mmv1/third_party/terraform/services/dataprocmetastore/resource_dataproc_metastore_federation_test.go

```diff
@@ -46,6 +45,33 @@ func TestAccMetastoreFederation_deletionprotection(t *testing.T) {
 	})
 }
 
+func TestAccMetastoreFederation_tags(t *testing.T) {
+	t.Parallel()
+
+	tagKey := acctest.BootstrapSharedTestTagKey(t, "dataproc_metastore_federation-tagkey")
+	context := map[string]interface{}{
+		"org":           envvar.GetTestOrgFromEnv(t),
+		"tagKey":        tagKey,
+		"tagValue":      acctest.BootstrapSharedTestTagValue(t, "dataproc_metastore_federation-tagvalue", tagKey),
+		"random_suffix": acctest.RandString(t, 10),
+	}
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMetastoreFederationTags(context),
+			},
+			{
+				ResourceName:            "google_dataproc_metastore_federation.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"tags"},
+			},
+		},
+	}),
```

Comment:
This is what's causing the gofmt errors.

```suggestion
	})
```